jython/InitAllTurnoutsSensors.py

```diff
@@ -24,6 +24,11 @@
 #######    WE HAVE LEARNED THE HARD WAY, THAT PANEL FILES THAT ACTIATE Turnouts
 #######    AT LOAD TIME, RISK THE CONSEQUENCES OF MOVING A Turnout UNDER A TRAIN
 
+# This script kicks off an independent series of actions what will take some time to 
```

Comment:
No need to change.
I'm not a native English speaker, but it seems to me that `what` should be `that`. But I might be wrong.
`series of actions what will take some time`